pywren/pywren_ibm_cloud/action/jobrunner.py

```diff
@@ -56,6 +58,11 @@ def b64str_to_bytes(str_data):
 storage_config = json.loads(os.environ.get('STORAGE_CONFIG', ''))
 internal_storage = storage.InternalStorage(storage_config)
 
+if 'SHOW_MEMORY_USAGE' in os.environ:
```

Comment:
@JosepSampe it's not suppose to be in configuration? otherwise, how it can get to os.environ?

Reply:
@gilv All you write in extra_env parameter goes to function environ.

pw.map(fun, iterdata, extra_env={'SHOW_MEMORY_USAGE': True})

Reply:
@JosepSampe ohh.. we activate it on the function level, with extra param. I guess it's ok, I originally thought it's more general addition that is enabled / disabled on the PyWren level.